### ISSUE TITLE
How to Deploy VATIC  in window?  

### ISSUE THREAD

**Issue by gsikheria** [cvondrick/vatic#36]
Hi there,

Is any one Deploy Vatic on windows?. Please help me if any one deployed it.

**Comment by mgserafi** [comment]
Has this been addressed?

**Comment by gsikheria** [comment]
No, VATIC is tested only on UBUNTU. If you are able to Deploy on Windows, pls share
